File: ryan-anderson/array-plus-array.js
function arrayPlusArray(arr1, arr2) {
    let sum1 = 0; //initialize variable for sum of entries in array 1
    let sum2 = 0; //initialize variable for sum of entries in array 1
    for (i = 0; i < arr1.length; i++) { //add entries of array 1 together
      sum1 += arr1[i];
    }
    for (i = 0; i < arr2.length; i++) { //add entries of array 2 together
      sum2 += arr2[i];
    }
    return sum1 + sum2 //add the two array sums
  }

File: ryan-anderson/day-28/array-plus-array.py
def array_plus_array(arr1,arr2):
    sum_one = 0
    sum_two = 0
    for i in arr1:
        sum_one += i
    for j in arr2:
        sum_two += j
    return sum_one + sum_two

#initialize sum variables for each array
#loop over each array adding each element to sum
#add the two sums together